integration/apps/pennant/pennant.py

```diff
@@ -43,21 +43,27 @@ def setup_run_args(parser):
     """ Add common arguments for all run scripts:
```

Comment:
I like this pattern of having a setup_run_args for the app.  We do need to be careful of collisions with the base scripts.  Perhaps a prefix like `--pennant-input`  and `--pennant-cores-per-node`?

Reply:
It was @bgeltz s idea :) I like it too. That's a good idea, I might have to make that change for other apps as well.

Reply:
Although, if there were conflicts argparse would crash anyway but I get the point...